src/tests/test_console.py

```diff
@@ -404,33 +404,41 @@ async def f():
         ).strip()
     )
 
-    assert (
-        exec_and_get_result(
-            dedent(
-                """
+    result = exec_and_get_result(
+        dedent(
+            """
             class Test:
                 def __repr__(self):
                     raise TypeError("hi")
 
             Test()
             """
-            ).strip()
-        )
-        == dedent(
-            f"""
+        ).strip()
+    )
+
+    assert result.startswith(
+        dedent(
+            f"""\
             >>> class Test:
             ...     def __repr__(self):
             ...         raise TypeError(\"hi\")
             ... \
 
             >>> Test()
             [[;;;terminal-error]Traceback (most recent call last):
-              File \"/lib/{PYVERSION}/site-packages/_pyodide/console.py\", line 465, in repr_shorten
+              File \"/lib/{PYVERSION}/site-packages/_pyodide/console.py\", line """
```

Comment:
I was thinking of something like `result = re.sub("line [0-9]*", "line xxx", result)` but I suppose this gets the job done too.

Reply:
I think doing the regex substitution is preferable because `assert x == y` gives better pytest error messages than `assert x.endswith(y)`.